backend/src/routes/api/namespaces/namespaceUtils.ts

```diff
@@ -16,6 +18,36 @@ export const applyNamespaceChange = (
       400,
     );
   }
+  const kc = fastify.kube.config;
+  const cluster = kc.getCurrentCluster();
+
+  const selfSubjectAccessReviewResult = await passThrough<V1SelfSubjectAccessReview>(
```

Comment:
Small nit, should we wanna try/catch it to log an internal error?

I was reviewing the passThrough method and the logic is covering most of the errors, but it seems off that we are not error handling this specific call.

Reply:
I think the pass-through function will log all the errors, so I don't make any other process here.